src/App.jsx

```diff
@@ -14,33 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
```

Comment:
```suggestion
```
you don't need state for visible goods.
all what to do by click on button just change actons state (it can be two separate states.)

make function which can be accept goodsFromServer, states and return visible goods.